blockchains/eth/lib/withdrawals_decoder.js

```diff
@@ -0,0 +1,36 @@
+const { computeGasExpense, computeGasExpenseBase36 } = require('./util');
+const constants = require('./constants');
+
+class WithdrawalsDecoder {
+  constructor(web3, web3Wrapper) {
+    this.web3Wrapper = web3Wrapper;
+  }
+
+  getWithdrawal(withdrawal, block, blockNumber) {
+    return [{
+      from: constants.BURN_ADDRESS,
```

Comment:
This is not correct, I guess we should set mint or 0x00..00 address

Reply:
Maybe use a brand new special address for that, like `ETH_WITHDRAWAL`, wdyt?

Reply:
sounds good